tests/end-to-end/upgrade/Gopkg.toml

```diff
@@ -3,10 +3,6 @@ required = [
   "golang.org/x/tools/cmd/goimports",
 ]
 
-[[constraint]]
```

Comment:
add constraint for `github.com/gofrs/uuid`

Reply:
Done!